templates/authentication-typescript/README.md

```diff
@@ -0,0 +1,98 @@
+# Brightlayer UI Authentication Template
+## React + TypeScript + Vite + Jest + Testing-library
+
+This project is a sample that demonstrates how to use [@brightlayer-ui/react-auth-workflow](https://www.npmjs.com/package/@brightlayer-ui/react-auth-workflow). It includes a sample Home screen with buttons to log out and change password. If the user is not authenticated, he/she will be presented with the authentication workflow with screens for Log In, Forgot Password, Support, and Registration.
+
+This project now includes an Okta login screen for authentication. To use the Okta login screen, you need to configure your Okta application and update the environment variables in your project.
+
+## Okta Configuration
+
+1. Create an application in Okta.
+2. Update the `.env` file in your project directory with the following variables:
+    ```
+    VITE_REACT_APP_OKTA_CLIENT_ID=<your-okta-client-id>
+    VITE_REACT_APP_OKTA_ISSUER=<your-okta-issuer-url>
+    ```
+**Note: The Okta login screen will be automatically presented to the user if they are not authenticated. The user can log in using their Okta credentials. Once authenticated, they will be redirected to the Home screen.**
+
+## This project was setup with Brightlayer-ui Packages
```

Comment:
Brightlayer UI is not hyphenated...do a search and replace to fix this throughout...I saw at least one other place where there was a hyphen.